src/status_im/contexts/chat/group_details/view.cljs

```diff
@@ -20,9 +20,7 @@
     :icon-only?          true
     :container-style     {:margin-left 20}
     :accessibility-label :back-button
-    :on-press            (fn []
-                           (rf/dispatch [:navigate-back])
-                           (rf/dispatch [:chat/close]))}
+    :on-press            #(rf/dispatch [:navigate-back])}
```

Comment:
We need to close currently active group chat or it'll keep `current-chat-id` in app-db causing unread messages to be set as read immediately (No unread indicator will be shown).

Reply:
Check [here](https://github.com/status-im/status-mobile/pull/18149)

Reply:
hi @ibrkhalil,
Thank you very much for reviewing PR.
This is what this PR is fixing. Its is removing opening of chat, just to check group details.

Reply:
I see, We no longer add current-chat-id when opening group details.
Thanks for the explanation Parvesh!